### DISCUSSION TITLE
How to configure floating layout for specific desktops / spaces / monitors?

### DISCUSSION THREAD

**Discussion by mickimnet** [ianyh/Amethyst#1459] [Q&A] [answered]
Hello everybody,

I would like to limit the tiling functionality to one monitor. Regarding #1381 and #1430, I should be able to set the floating layout for specific spaces / desktops, but I didn't find any documentation / configuration option to do that.

Therefore, how do I set the floating layout for spaces / desktops?

Thanks and take care
Mick

**Comment by ianyh** [comment, marked as answer]
You can switch to that layout on the relevant space. Subsequent launches of Amethyst will try its best to recover those selections, though it doesn't currently persist that state when the app crashes.

**Reply by mickimnet** [reply]
Thank you for the information. Will try it out. Do you plan to add a feature to explicitly select it in the configuration?

**Reply by ianyh** [reply]
Not immediately, but I'm not opposed to it. I think there might already be an issue tracking it.

**Reply by mickimnet** [reply]
Are you referring to #1430 which is still open? If so, I would add the suggestion to it and change the tag from _Question_ to _Enhancement_.

**Reply by ianyh** [reply]
No, I'm referring to #302.
